### ISSUE TITLE
2 plugins not working

### ISSUE THREAD

**Issue by mercadanteus** [PowerNukkit/PowerNukkit#1145]
Checking logs I found this 2 plugins failing to load, could you take a look?  https://hastebin.com/patasilemi.properties

### ▶ Steps to Reproduce
Install Antixray and DogImage plugins, and run server
DogImage: https://cloudburstmc.org/resources/dogimage-any-image.496/
AntiXray:  https://github.com/wode490390/AntiXray

### 📋 Debug information

* PowerNukkit version: ✍
* Debug link: ✍
* Timings link (if relevant): ✍

**Comment by mercadanteus** [comment]
> * PowerNukkit version: ✍
> * Debug link: ✍
> * Timings link (if relevant): ✍

**Comment by joserobjr** [comment]
Unfortunately, AntiXray is impossible to make it compatible because of the way that it was implemented. It needs to be adjusted to do not use reflections to access private stuff to be compatible.

I opened a plugin request thread: https://discuss.powernukkit.org/t/request-port-of-antixray-plugin/273

DogImage is new to me. Can you give me a plugin configuration that reproduces this bug? I tried with the default one but it started normally

**Comment by joserobjr** [comment]
The startup issue didn't happen for me, but I did find a compatibility issue. It placed the item frames with incorrect meta:
![image](https://user-images.githubusercontent.com/3679022/123278709-8938e400-d4dd-11eb-81d4-dd17b1eb7ca3.png)

**Comment by mercadanteus** [comment]
Hello and thanks, using 1.5-PN-MC1.17.0-B64 DogImage loads indeed, but misplace frames as you said, using nukkitx worked fine, I don´t know if something can be done in PM side, as contact chinese developer can be a hard task
![123278709-8938e400-d4dd-11eb-81d4-dd17b1eb7ca3](https://user-images.githubusercontent.com/24380156/123313364-16386900-d4ef-11eb-9cb5-fc5cee8b978d.png)

**Comment by joserobjr** [comment]
> I hope something can be done in PN side, as contact chinese developer can be a hard task.

Yes, that's definitely a thing on our end, and I know exactly what is causing it. Don't contact the plugin devs on this issue. It might affect other plugins that set numeric meta values directly to the item frame blocks.
